server/src/main/java/io/druid/server/AsyncManagementForwardingServlet.java

```diff
@@ -19,6 +19,7 @@
 
 package io.druid.server;
 
+import com.amazonaws.thirdparty.apache.http.client.utils.URIBuilder;
```

Comment:
Should this really be using the shaded one?

Reply:
Nice catch! Fixed it.